blivet/formats/fs.py

```diff
@@ -168,6 +173,18 @@ def dict(self):
                   "mountable": self.mountable})
         return d
 
+    @classmethod
```

Comment:
Why is this a class method? I'd just make it a @property and use self.size instead of passing it in.